### DISCUSSION TITLE
Determine running version

### DISCUSSION THREAD

**Discussion by mart3323** [excalidraw/excalidraw#6455] [Q&A] [answered]
## Questions
1. Is there a way to determine the currently running version of the app (and ideally when it was last updated?)
2. Is there anywhere i can see the changelogs/release notes for the live app?

## Background
Starting this morning (approx 7 hours ago), when i select an object and hit <kbd>S</kbd> <kbd>1/2/3</kbd> to change the stroke color, the <kbd>S</kbd> key correctly opens the color menu but trying to select a color will not work and will trigger other shortcuts in the background instead. For example the numbers (and A and D) switch tools while keeping the color menu open

Before jumping to report it as a bug (or maybe even trying to trace down the cause myself), i would like to check if there's actually been a recent update, because if there hasn't and it was working yesterday then clearly it must be something wrong on my end. 
In most applications there's an 'about' section that has the current version, but i was not able to find anything like that in excalidraw. And in the actions tab i don't know which pipelines update the live application and which are just tests or nightly builds

**Comment by dwelle** [comment, marked as answer]
Hi!

> Is there a way to determine the currently running version of the app (and ideally when it was last updated?)

Right-click on canvas -> stats for nerds -> click on version to copy to clipboard

> Is there anywhere i can see the changelogs/release notes for the live app?

We don't have a changelog for the app yet. So the best bet is to scroll through he [commit log](https://github.com/excalidraw/excalidraw/commits/master). Features are prefix with `feat:` (though not all `feat:` are necessarily fully user-facing features).

> Starting this morning (approx 7 hours ago), when i select an object and hit S 1/2/3 to change the stroke color, the S key correctly opens the color menu but trying to select a color will not work and will trigger other shortcuts in the background instead. For example the numbers (and A and D) switch tools while keeping the color menu open

Oh, you're right. Will create an issue for this for you!

**Reply by dwelle** [reply]
Actually was faster to fix it directly: https://github.com/excalidraw/excalidraw/pull/6464. Will release shortly.
